controllers/api/v1/search_test.go

```diff
@@ -0,0 +1,318 @@
+package apiv1
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/google/uuid"
+
+	_ "google-scraper/initializers"
+	. "google-scraper/tests"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("SearchController", func() {
+	Describe("POST /api/v1/search", func() {
+		Context("Given the valid credential", func() {
```

Comment:
`the` is kind of redundant here:

```suggestion
		Context("Given valid credentials", func() {
```